alembic/versions/e29e1b255d90_create_idx_proj_task_result_warm_cache.py

```diff
@@ -0,0 +1,26 @@
+"""create indexes project task result for warm_cache
+
+Revision ID: e29e1b255d90
+Revises: 2edf951cc6ae
+Create Date: 2018-09-24 11:40:56.979019
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e29e1b255d90'
+down_revision = '2edf951cc6ae'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_index('result_project_id_info_idx', 'result', ['project_id', 'info'])
```

Comment:
`project_id` should be already indexed. I am also not convinced of the benefits of indexing `info`.

Reply:
This was for pybossa.cache.projects.n_results. Execution time before and after index were
Execution time: 7.879 ms(before)
Execution time: 0.085 ms(after)

Reply:
Other queries are impacted, inserts in particular. Also I think there is a size limit on indexed data, if the info exceeds it insert will fail.

Reply:
agree. info field could have much higher size and indexing would fail. since n_result is not utilized in workflow as such, we can remove computing it for now and check how the warm up cache performs. i'll make the changes. thanks.